src/utilities/replay/replay.cpp

```diff
@@ -197,16 +257,16 @@ main(int argc, char *argv[])
   conduit::Node replay_data;
   //replay_data.print();
   conduit::Node ascent_opts;
-  ascent_opts["actions_file"] = options.m_actions_file;
   ascent_opts["ascent_info"] = "verbose";
 #if defined(ASCENT_REPLAY_MPI)
   ascent_opts["mpi_comm"] = MPI_Comm_c2f(MPI_COMM_WORLD);
 #endif
 
   //
-  // Blank actions to be populated from actions file
+  // Populate actions with actions file
   //
-  conduit::Node actions;
+  int mpi_comm = ascent_opts.has_child("mpi_comm") ? ascent_opts["mpi_comm"].to_int() : 0;
```

Comment:
Should the case were we don't have a comm be `-1`, to trigger the right error conditions?